src/components/resources/ResourcesOverview.js

```diff
@@ -5,15 +5,46 @@ import {useSelector, useDispatch} from 'react-redux';
 import {bindActionCreators} from 'redux';
 import {actionCreators} from '../../state';
 import ResourcesView from '../tooltip/ResourcesView';
-
+import {useEffect} from 'react';
 
 export const ResourcesOverview = () => {
   const currentUserResources = useSelector(state => state.currentUserResources);
-  const {setTooltipResources} = bindActionCreators(actionCreators, useDispatch());
+  const {setTooltipResources, clearTooltipResources} = bindActionCreators(actionCreators, useDispatch());
   const tooltipResourcesView = useSelector(state => state.tooltipResourcesView);
 
-  const handleClick = currentResource => {
-    setTooltipResources(!tooltipResourcesView.status[currentResource.name], currentResource.name, currentResource);
+  useEffect(() => {
+    for (const key in tooltipResourcesView.status) {
+      if (tooltipResourcesView.status[key]) {
+        window.addEventListener('click', handleClick, false);
+
+        return () => {
+          window.removeEventListener('click', handleClick, false);
+        };
+      }
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [tooltipResourcesView]);
+
+  function handleClick(event) {
+    event.stopPropagation();
+
+    for (const key in tooltipResourcesView.status) {
+      if (tooltipResourcesView.status[key] && tooltipResourcesView.click === key) {
+        setTooltipResources(tooltipResourcesView.status[key], key, tooltipResourcesView.currentResources.value);
+        break;
+      } else {
+        clearTooltipResources(false);
+      }
+    }
+  }
+
+  const handleClickOpenResourcenView = currentResource => {
+    setTooltipResources(
+      !tooltipResourcesView.status[currentResource.name],
+      currentResource.name,
+      currentResource,
+      currentResource.name
```

Comment:
Why do you use the `name` key twice?

Reply:
i have given the function setTooltipResources a 4th optional parameter, this serves as a reference to which resource you have clicked on